requirements/requirements_doc.txt

```
@@ -8,9 +8,9 @@ numpy==2.2.3
 openpyxl==3.1.5
 osmnx==2.0.2
 pandas==2.2.3
-pyvista==0.44.2
-fast-simplification==0.1.9
-joblib==1.4.2
+pyvista==0.45.0
```

```suggestion
pyvista==0.44.2
```
We can't yet use 0.45.0. Some extra changes in CICD are required to work correctly.